articles/connect-a-publish-branch.mdx

```diff
@@ -0,0 +1,70 @@
+---
+_schema: default
+_uuid: 3536db99-3aa9-4978-bada-6bc16aaf8e87
+_created_at: '2021-09-24 00:53:28 +0000'
+title: Connect a Publish Branch
+nav_title: Connect a Publish Branch
+published: true
+image: /documentation/static/CloudCannonDocumentationog.jpg
+description: >-
+  Learn how to set up a publish branch in CloudCannon, using immediate merging
+  or pull requests.
+tags:
+  - branch
+  - git
+  - publish
+related_articles:
+  - articles/merging-and-pull-requests.mdx
+  - articles/create-a-project.mdx
+related_links:
+---
+The CloudCannon publishing workflow lets your team edit your content on a branch site. Team members can merge their changes into the Publish Branch directly or with a pull request.
+
+Connecting a Publish Branch allows you to create a staging site to test changes without affecting your live site.
+
+* **Publish Branch** — This is the site to which you publish your changes. This could be your live site.
+* **Branch Site** — This is the site on which you test your changes. This could be your staging site.
```

Comment:
Could be "on which you make or test your changes" — allows for the changes to be either development or content.